pkg/config/cmd_terminal.go

```diff
@@ -32,12 +34,12 @@ var (
 		MemoryLimit: "256Mi",
 		Container: devworkspace.Container{
 			Name:  "dev",
-			Image: "registry.redhat.io/codeready-workspaces/plugin-openshift-rhel8:2.1",
+			Image: images.GetWebTerminalToolingImage(),
```

Comment:
Now it can be "" which will lead deployment to fail. Maybe it's better to fail workspace start in such case.

Reply:
Either way the deployment is broken; if that env var resolves to "" the only thing that will realistically fix it is a respin + re-release. These aren't configuration properties so them not being set should never reach production.

I've set it to fail workspace startup if this env var is unset. Going forward though -- if *any* of the related_images envrionment variables is unset it will break the controller. Do we need to validate each of them?